src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/model/StaffDetail.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.restapi.model
+
+import java.util.UUID
+
+data class StaffDetail(
+  val staffId: UUID,
+  val firstName: String,
+  val lastName: String,
+  val primaryEmail: String,
+  val username: String,
+  val type: String,
```

Comment:
If you move the PomType Enum out of the Entity implementation, it could also be used here in the model class. Or even just referenced.